.eslintrc.js

```diff
@@ -106,6 +106,7 @@ module.exports = {
     'valid-typeof': 'off',
     'no-undef': 'off',
     'no-return-assign': 'off',
+    'no-fallthrough': ['error', { commentPattern: 'fallthrough' }],
```

Comment:
Where do we need this exception to the rule?

Reply:
No longer applicable. Will remove